charts/cortx/values.yaml

```diff
@@ -219,6 +219,10 @@ control:
   enabled: true
   # -- Number of Control replicas
   replicaCount: 1
+  # -- Name of Secret that contains the certificate
+  certificateSecret: ""
+  # -- Name of Secret key that contains the certificate
+  certificateSecretKey: ""
```

Comment:
I like the idea of being able to specify these separately, but I don't think CORTX allows this? The cluster configuration only provides a single certificate value for all components. What would happen if you actually used two different secrets?

I think this has to be a single value at the top-level.